src/components/landing-page/LandingPage.jsx

```diff
@@ -56,18 +55,15 @@ class LandingPage extends React.Component {
         const { classes } = this.props;
         const items = [
             ["instance-configurator", i18n.t("Instance Configuration"), "edit", GRID_ROW_1],
-            ["sync/organisationUnits", i18n.t("Organisation Units"), "sync", GRID_ROW_3],
-            ["sync/dataElements", i18n.t("Data Elements"), "sync", GRID_ROW_3],
-            ["sync/indicators", i18n.t("Indicators"), "sync", GRID_ROW_3],
-            ["sync/validationRules", i18n.t("Validation Rules"), "sync", GRID_ROW_3],
+            ["sync/metadata", i18n.t("Metadata"), "sync", GRID_ROW_2],
```

Comment:
I guess it should be "Metadata synchronization".

Reply:
The old design has no synchronization word in sync links, for this reason, I have not added it
![Old_design](https://user-images.githubusercontent.com/5593590/66809895-82975280-ef2e-11e9-8bdf-54c9465695bc.png)

Reply:
Yeah originally we had Organisation Unit synchronization but it felt too much x4, but maybe now it's more significative since we only have one label for it.

Anyway @adrianq will have the final word.

Reply:
@SferaDev @xurxodev Not a strong preference, I think it looks a bit better 'Metadata Synchronization' so I have changed it. Anyway, I have the feeling we will change it again before releasing the first version